ruggedpod_api/__init__.py

```diff
@@ -14,3 +14,16 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+import os.path
+
+from ruggedpod_api.common.conf import YmlConf
+
+CONF_FILE = "/etc/ruggedpod/api-conf.yaml"
```

Comment:
There's currently no documentation in the README. By the way, the legacy behaviour is still working.